internals/scripts/build.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
```

Comment:
One small nit, usually node scripts have the following `#!/usr/bin/env node` on the first line, so that OS can detect node as an interpreter. And it is also nice to quickly distinguish between configuration .js files and/or source files.

Reply:
I don't see any benefit of using it here.